### PR TITLE
adding bucket for goldens for web builds

### PR DESCRIPTION
**Pull request by nturgut**
We were missing the bucket name for [web_engine recipe](https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/web_engine.py#207) to use in order to upload failed golden results.

Current test results: https://ci.chromium.org/p/flutter/builders/try/Linux%20Web%20Engine/7164

When we manually add the `gcs_golden_bucket` property with `led` https://chromium-swarm.appspot.com/task?id=4d91920a41eba210 

flutter_logs is a public visible bucket.

fixes: https://github.com/flutter/flutter/issues/62077

### CODE REVIEW

**Comment by nturgut** [comment]
@christopherfujino I wasn't sure how to generate the config file. I'll use `lucicfg` from depot_tools. What parameters should I add?

**Comment by nturgut** [comment]
> @christopherfujino I wasn't sure how to generate the config file. I'll use `lucicfg` from depot_tools. What parameters should I add?

Nw! Cirrus gave an error with a log :)

File: config/engine_config.star [hunk 1]
```diff
@@ -162,6 +165,7 @@ def engine_prod_config(platform_args, branch, version, ref, fuchsia_ctl_version)
     common.linux_prod_builder(
         name = builder_name("Linux%s Web Engine|lwe", branch),
         recipe = full_recipe_name("web_engine", version),
+        properties = engine_properties(gcs_goldens_bucket = "flutter_logs"),
```
Comment (godofredoc) [inline config/engine_config.star:168]: This is only for prod builders are we missing the try configs?

**Comment by godofredoc** [comment]
> erate the config file. I'll use `lucicfg` from depot_tools.

```
lucicfg fmt .
lucicfg lint .
```

File: config/engine_config.star (hunk 1 above)
Comment (nturgut) [reply config/engine_config.star:168]: done. thanks!

**Comment by nturgut** [comment]
Thanks for the quick review!

**Pull request merged**